IHP/Mail.hs

```diff
@@ -19,10 +19,14 @@ import IHP.FrameworkConfig
 
 import           Network.Mail.Mime
 import qualified Network.Mail.Mime.SES                as Mailer
+import qualified Network.Mail.SMTP                    as SMTP
 import qualified Network.HTTP.Client
 import qualified Network.HTTP.Client.TLS
 import Text.Blaze.Html5 (Html)
 import qualified Text.Blaze.Html.Renderer.Text as Blaze
+import qualified Data.ByteString.Lazy.Char8 as B8
```

Comment:
```suggestion
import qualified Data.ByteString.Lazy.Char8 as LazyByteString
```

Reply:
The IHP code mostly follows these conventions when using qualified imports. To make the code feel consistent I suggest changing this :)